Polygon_mesh_processing/test/Polygon_mesh_processing/CMakeLists.txt

```diff
@@ -76,6 +76,7 @@ if (EIGEN3_FOUND)
   create_single_source_cgal_program("remeshing_test.cpp" )
   create_single_source_cgal_program("measures_test.cpp")
   create_single_source_cgal_program("triangulate_faces_test.cpp")
+  create_single_source_cgal_program("test_pmp_remove_border_edge.cpp")
```

Comment:
Sébastien, that issue is rather trivial to fix: either add/commit/push the file, or remove its reference in the `CMakeLists.txt`.

Maybe you did not see the mail.